cmd/manifest/manifest.go

```diff
@@ -4,83 +4,92 @@ import (
 	"encoding/json"
 	"fmt"
 	"github.com/skupperproject/skupper/pkg/images"
-	"github.com/spf13/cobra"
 	"os"
+	"os/exec"
+	"strings"
 )
 
 type SkupperImage struct {
 	Name       string `yaml:"name"`
+	SHA        string `yaml:"sha"`
 	Repository string `yaml:"repository,omitempty"`
 }
 
-func generateManifestFile() error {
+func getSha(imageName string) string {
```

Comment:
Maybe suggest moving to pkg/images/image_utils.go

Reply:
Updated in 3d483631899741a0ec21919c4afb0a908b51fb77